api/src/main/scala/hmda/api/http/institutions/submissions/SubmissionEditPaths.scala

```diff
@@ -46,18 +46,33 @@ trait SubmissionEditPaths
           completeVerified(institutionId, period, seqNr, uri) {
             val fEditChecks = getValidationState(institutionId, period, seqNr)
 
-            val fSummaryEdits = fEditChecks.map { editChecks =>
-              val s = validationErrorsToEditResults(editChecks.tsSyntactical, editChecks.larSyntactical, Syntactical)
-              val v = validationErrorsToEditResults(editChecks.tsValidity, editChecks.larValidity, Validity)
-              val q = validationErrorsToEditResults(editChecks.tsQuality, editChecks.larQuality, Quality)
-              val m = validationErrorsToMacroResults(editChecks.larMacro)
-              SummaryEditResults(s, v, q, m)
-            }
+            parameters("sortBy".?) {
+              case Some("row") =>
+                val fRowSummary: Future[RowResults] = fEditChecks.map { e =>
+                  val tsErrors = e.tsSyntactical ++ e.tsValidity ++ e.tsQuality
+                  val larErrors = e.larSyntactical ++ e.larValidity ++ e.larQuality
+                  val macroErrors = e.larMacro
+                  validationErrorsToRowResults(tsErrors, larErrors, macroErrors)
```

Comment:
I think the creation of these three fields (`tsErrors`, `larErrors`, and `macroErrors`) could be moved inside the `validationErrorsToRowResults` function.  It would result in more concise code here, and a smaller method signature.